src/main/java/frc/robot/hardware/request/angle/IAngleRequest.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.hardware.request.angle;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public interface IAngleRequest {
```

Comment:
maby make another interface for position (linear position in meters) or is it the IValueRequest

Reply:
its the ivalue, now it was made for volt and current